### DISCUSSION TITLE
Welcome to Student_Survivor Discussions!

### DISCUSSION THREAD

**Discussion by YOON-JAEYEONG** [YOON-JAEYEONG/Student_Survivor#17] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rlatldnd1** [comment]
하이

**Comment by rlatldnd1** [comment]
범위공격과 콜라이더 구현에 대해
범위 공격의 콜라이더는 : 이후 변경되는 콜라이더는 해당 Bullet의 자식 오브젝트로 구현합시다.
소스 코드 상에서도 getcomponentsInchild?로 바로 쓸 수 있고
단계에 따라 자식 콜라이더 여러 명 만들면 편할 듯

이렇게 되면 레벨업 시에 증가하는 콜라이더 범위를 쉽게 구현 가능
+ 스킬_운체 : 달려가는 폭탄 처럼 런타임 중에 변경되는 콜라이더도 쉽게 구현 가능
